src/components/__tests__/markdown.test.js

```diff
@@ -0,0 +1,233 @@
+import React from "react"
+import renderer from "react-test-renderer"
```

Comment:
Please rely on `enzyme-to-json` library and do not use `react-test-renderer`  - there is no trace of it in `master` now. Can see [here](https://github.com/rahuldkjain/github-profile-readme-generator/blob/master/src/components/__tests__/work.test.js#L22)